### PR TITLE
Added workflow to use FFMPEG to convert video to gif

### PR DESCRIPTION
**Pull request by olsoma13**
## Discord username (optional) please include so we can attribute you with our Contributor role (like so elvis#4747)
molson#0565
## Description of changes (updated or new workflows)
New workflow to convert videos to gifs

### CODE REVIEW

File: specs/ffmpeg/vid_to_gif.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+name: ffmpeg to convert a video to gif
```
Comment (alokedesai) [inline specs/ffmpeg/vid_to_gif.yml:2]: How about "Convert a video to gif using ffmpeg"?

File: specs/ffmpeg/vid_to_gif.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  - name: start
+    description: Start time in seconds of your input media file.
+    default_value: 0.0
+  - name: length
+    description: Output duration in seconds.
+    default_value: 2.5
+  - name: scale
+    description: output scale defined pixels (integer). Will maintain aspect ratio.
+    default_value: 1080
+source_url: "https://github.com/cyburgee/ffmpeg-guide"
```
Comment (alokedesai) [inline specs/ffmpeg/vid_to_gif.yml:19]: nit: could we link to the actual section of this guide where it talks about converting from video to gif?
https://github.com/cyburgee/ffmpeg-guide#video-to-gif

**Review by alokedesai — CHANGES_REQUESTED** [review]
Looks great, thank you for contributing! I had a few small comments but nothing major :D

**Comment by elviskahoro** [comment]
@olsoma13 just reaching back out to see if you could make a few small changes based on aloke's feedback once you have free cycles

**Comment by elviskahoro** [comment]
Closing for now, please re-open if you get the bandwidth to resolve the comments.

**Pull request closed without merging**